docs/installation_guide.md

```diff
@@ -210,6 +219,20 @@ $ sudo prepare --with-notary
 $ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.notary.yml up -d
 ```
 
+#### _Managing lifecycle of Harbor when it's installed with Clair_ 
+
+When Harbor is installed with Clair_, an extra template file ```docker-compose.clair.yml``` is needed for docker-compose commands. The docker-compose commands to manage the lifecycle of Harbor are:
+```
+$ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.clair.yml [ up|down|ps|stop|start ]
+```
+For example, if you want to change configuration in ```harbor.cfg``` and re-deploy Harbor when it's installed with Clair, the following commands should be used:
+```sh
+$ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.clair.yml down -v
+$ vim harbor.cfg
+$ sudo prepare --with-clair
+$ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.clair.yml up -d
+```
+
```

Comment:
If you have installed Notary and Clair, you should include both components in the docerk-compose and prepare commands:

give the commands.